bundles/tools.mdsd.modelingfoundations.identifier/META-INF/MANIFEST.MF

```diff
@@ -7,11 +7,12 @@ Bundle-Version: 0.1.0.qualifier
 Bundle-ClassPath: .
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
-Bundle-RequiredExecutionEnvironment: JavaSE-10
+Bundle-RequiredExecutionEnvironment: JavaSE-11
 Export-Package: tools.mdsd.modelingfoundations.identifier,
  tools.mdsd.modelingfoundations.identifier.impl,
  tools.mdsd.modelingfoundations.identifier.util
 Require-Bundle: org.eclipse.emf.ecore;visibility:=reexport,
  org.eclipse.core.runtime,
- org.eclipse.emf.cdo
+ org.eclipse.emf.cdo,
+ tools.mdsd.modelingfoundations.workflow;bundle-version="0.1.0";resolution:=optional
```

Comment:
Why is there a dependency from the model bundle to the workflow bundle? Even if the resolution is optional, I do not see why this is necessary.

Reply:
The optional dependency between model and workflow is required for tycho to order the build the workflow plugin ahead of the model plugin.
Alternatively, we could get rid of the workflow bundle entirely and merge it into the model bundle.

Reply:
Actually I like the separation of workflow and meta model. Other options to enforce the order in the reactor are either more error prone or more complex. Therefore, this seems to be the best solution.